analyzer/src/main/kotlin/managers/NuGet.kt

```diff
@@ -95,3 +104,59 @@ class NuGetPackageFileReader : XmlPackageFileReader {
         return ids
     }
 }
+
+/**
+ * A reader for XML-based NuGet configuration files, see
+ * https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file
+ */
+class NuGetConfigFileReader {
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    @XmlRootElement(name = "configuration")
+    private data class NuGetConfig(
+        // Contains key:value pairs as well as the `<clear />` tag to ignore previously defined configuration values.
```

Comment:
I mentioned it because the implementation cannot use a `List<PackageSource>` due to possible `<clear />` tags.
But you're right, I cannot be certain that a `<clear />` tag will be present and in fact won't be present in most cases.
I would suggest to remove this comment or would you like me to reword it?

Reply:
Just removing it is fine with me.